lib/http/HttpRequestApple.hpp

```diff
@@ -0,0 +1,34 @@
+//
```

Comment:
This file should be named HttpRequest_Apple.http (similar case with new .mm file) to match current platform specific naming scheme.